### ISSUE TITLE
HW2 db가 비어 있을 때 add 질문

### ISSUE THREAD

**Issue by cki86201** [swsnu/swppfall2018#108]
angular-in-memory-web-api 가 만들어 주는 id를 사용하면 된다고 하셨습니다.

그런데 찾아 보니 angular-in-memory-web-api는 db에 아무 것도 없을 경우 새로 추가되는 item의 id를 결정할 때 에러가 발생하는 것 같습니다.
[https://github.com/angular/angular/issues/22120](url)

이런 경우는 처리하지 않아도 되나요? 즉, 글이 없을 때 글을 쓰는 동작을 할 수 없어도 괜찮나요?

감사합니다.

**Comment by sanha** [comment]
네, memory DB specific한 문제로 보이네요. 모두 삭제하고 다시 글을 쓰는 등의 테스트를 하지 않겠습니다.
